Add label for Droplr

lucascantor opened the pull request:
```
sudo utils/assemble.sh droplr DEBUG=0
2024-04-06 22:20:29 : REQ   : droplr : ################## Start Installomator v. 10.6beta, date 2024-04-06
2024-04-06 22:20:29 : INFO  : droplr : ################## Version: 10.6beta
2024-04-06 22:20:29 : INFO  : droplr : ################## Date: 2024-04-06
2024-04-06 22:20:29 : INFO  : droplr : ################## droplr
2024-04-06 22:20:29 : DEBUG : droplr : DEBUG mode 1 enabled.
2024-04-06 22:20:29 : INFO  : droplr : SwiftDialog is not installed, clear cmd file var
2024-04-06 22:20:30 : INFO  : droplr : setting variable from argument DEBUG=0
2024-04-06 22:20:30 : DEBUG : droplr : name=Droplr
2024-04-06 22:20:31 : DEBUG : droplr : appName=
2024-04-06 22:20:31 : DEBUG : droplr : type=dmg
2024-04-06 22:20:31 : DEBUG : droplr : archiveName=
2024-04-06 22:20:31 : DEBUG : droplr : downloadURL=https://github.com/Droplr/droplr-desktop-releases/releases/download/v1.7.0/Droplr-1.7.0-arm64.dmg
2024-04-06 22:20:31 : DEBUG : droplr : curlOptions=
2024-04-06 22:20:31 : DEBUG : droplr : appNewVersion=1.7.0
2024-04-06 22:20:31 : DEBUG : droplr : appCustomVersion function: Not defined
2024-04-06 22:20:31 : DEBUG : droplr : versionKey=CFBundleShortVersionString
2024-04-06 22:20:31 : DEBUG : droplr : packageID=
2024-04-06 22:20:31 : DEBUG : droplr : pkgName=
2024-04-06 22:20:31 : DEBUG : droplr : choiceChangesXML=
2024-04-06 22:20:31 : DEBUG : droplr : expectedTeamID=MZ25PHMY7Y
2024-04-06 22:20:31 : DEBUG : droplr : blockingProcesses=Droplr Droplr Helper (Plugin) Droplr Helper (GPU) Droplr Helper (Renderer) Droplr Helper
2024-04-06 22:20:31 : DEBUG : droplr : installerTool=
2024-04-06 22:20:31 : DEBUG : droplr : CLIInstaller=
2024-04-06 22:20:31 : DEBUG : droplr : CLIArguments=
2024-04-06 22:20:31 : DEBUG : droplr : updateTool=
2024-04-06 22:20:31 : DEBUG : droplr : updateToolArguments=
2024-04-06 22:20:31 : DEBUG : droplr : updateToolRunAsCurrentUser=
2024-04-06 22:20:31 : INFO  : droplr : BLOCKING_PROCESS_ACTION=tell_user
2024-04-06 22:20:31 : INFO  : droplr : NOTIFY=success
2024-04-06 22:20:31 : INFO  : droplr : LOGGING=DEBUG
2024-04-06 22:20:31 : INFO  : droplr : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2024-04-06 22:20:31 : INFO  : droplr : Label type: dmg
2024-04-06 22:20:31 : INFO  : droplr : archiveName: Droplr.dmg
2024-04-06 22:20:31 : DEBUG : droplr : Changing directory to /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.DuHitxN11u
2024-04-06 22:20:31 : INFO  : droplr : name: Droplr, appName: Droplr.app
2024-04-06 22:20:31.439 mdfind[11959:2710132] [UserQueryParser] Loading keywords and predicates for locale "en_US"
2024-04-06 22:20:31.439 mdfind[11959:2710132] [UserQueryParser] Loading keywords and predicates for locale "en"
2024-04-06 22:20:31.496 mdfind[11959:2710132] Couldn't determine the mapping between prefab keywords and predicates.
2024-04-06 22:20:31 : WARN  : droplr : No previous app found
2024-04-06 22:20:31 : WARN  : droplr : could not find Droplr.app
2024-04-06 22:20:31 : INFO  : droplr : appversion: 
2024-04-06 22:20:31 : INFO  : droplr : Latest version of Droplr is 1.7.0
2024-04-06 22:20:31 : REQ   : droplr : Downloading https://github.com/Droplr/droplr-desktop-releases/releases/download/v1.7.0/Droplr-1.7.0-arm64.dmg to Droplr.dmg
2024-04-06 22:20:31 : DEBUG : droplr : No Dialog connection, just download
2024-04-06 22:20:55 : DEBUG : droplr : File list: -rw-r--r--  1 root  wheel   164M Apr  6 22:20 Droplr.dmg
2024-04-06 22:20:55 : DEBUG : droplr : File type: Droplr.dmg: zlib compressed data
2024-04-06 22:20:55 : DEBUG : droplr : curl output was:
*   Trying 140.82.112.3:443...
* Connected to github.com (140.82.112.3) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
} [315 bytes data]
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (IN), TLS handshake, Server hello (2):
{ [122 bytes data]
* (304) (IN), TLS handshake, Unknown (8):
{ [19 bytes data]
* (304) (IN), TLS handshake, Certificate (11):
{ [3137 bytes data]
* (304) (IN), TLS handshake, CERT verify (15):
{ [79 bytes data]
* (304) (IN), TLS handshake, Finished (20):
{ [36 bytes data]
* (304) (OUT), TLS handshake, Finished (20):
} [36 bytes data]
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=github.com
*  start date: Mar  7 00:00:00 2024 GMT
*  expire date: Mar  7 23:59:59 2025 GMT
*  subjectAltName: host "github.com" matched cert's "github.com"
*  issuer: C=GB; ST=Greater Manchester; L=Salford; O=Sectigo Limited; CN=Sectigo ECC Domain Validation Secure Server CA
*  SSL certificate verify ok.
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://github.com/Droplr/droplr-desktop-releases/releases/download/v1.7.0/Droplr-1.7.0-arm64.dmg
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: github.com]
* [HTTP/2] [1] [:path: /Droplr/droplr-desktop-releases/releases/download/v1.7.0/Droplr-1.7.0-arm64.dmg]
* [HTTP/2] [1] [user-agent: curl/8.4.0]
* [HTTP/2] [1] [accept: */*]
> GET /Droplr/droplr-desktop-releases/releases/download/v1.7.0/Droplr-1.7.0-arm64.dmg HTTP/2
> Host: github.com
> User-Agent: curl/8.4.0
> Accept: */*
> 
< HTTP/2 302 
< server: GitHub.com
< date: Sun, 07 Apr 2024 05:20:31 GMT
< content-type: text/html; charset=utf-8
< vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
< location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/595150466/7ebc619a-5088-42b2-b90a-475348892192?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240407%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240407T052031Z&X-Amz-Expires=300&X-Amz-Signature=4e3f6efff3f127efede8c27ef88fb193a51f757ffd3e72bcabc6d925e0247f7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=595150466&response-content-disposition=attachment%3B%20filename%3DDroplr-1.7.0-arm64.dmg&response-content-type=application%2Foctet-stream
< cache-control: no-cache
< strict-transport-security: max-age=31536000; includeSubdomains; preload
< x-frame-options: deny
< x-content-type-options: nosniff
< x-xss-protection: 0
< referrer-policy: no-referrer-when-downgrade
< content-security-policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com api.githubcopilot.com objects-origin.githubusercontent.com *.actions.githubusercontent.com wss://*.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ productionresultssa10.blob.core.windows.net/ productionresultssa11.blob.core.windows.net/ productionresultssa12.blob.core.windows.net/ productionresultssa13.blob.core.windows.net/ productionresultssa14.blob.core.windows.net/ productionresultssa15.blob.core.windows.net/ productionresultssa16.blob.core.windows.net/ productionresultssa17.blob.core.windows.net/ productionresultssa18.blob.core.windows.net/ productionresultssa19.blob.core.windows.net/ github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com copilot-workspace.githubnext.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com gist.github.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
< content-length: 0
< x-github-request-id: E87E:240776:502AB3:684F98:66122D1F
< 
{ [0 bytes data]
* Connection #0 to host github.com left intact
* Issue another request to this URL: 'https://objects.githubusercontent.com/github-production-release-asset-2e65be/595150466/7ebc619a-5088-42b2-b90a-475348892192?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240407%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240407T052031Z&X-Amz-Expires=300&X-Amz-Signature=4e3f6efff3f127efede8c27ef88fb193a51f757ffd3e72bcabc6d925e0247f7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=595150466&response-content-disposition=attachment%3B%20filename%3DDroplr-1.7.0-arm64.dmg&response-content-type=application%2Foctet-stream'
*   Trying 185.199.109.133:443...
* Connected to objects.githubusercontent.com (185.199.109.133) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
} [334 bytes data]
* (304) (IN), TLS handshake, Server hello (2):
{ [122 bytes data]
* (304) (IN), TLS handshake, Unknown (8):
{ [19 bytes data]
* (304) (IN), TLS handshake, Certificate (11):
{ [3099 bytes data]
* (304) (IN), TLS handshake, CERT verify (15):
{ [264 bytes data]
* (304) (IN), TLS handshake, Finished (20):
{ [36 bytes data]
* (304) (OUT), TLS handshake, Finished (20):
} [36 bytes data]
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: C=US; ST=California; L=San Francisco; O=GitHub, Inc.; CN=*.github.io
*  start date: Mar 15 00:00:00 2024 GMT
*  expire date: Mar 14 23:59:59 2025 GMT
*  subjectAltName: host "objects.githubusercontent.com" matched cert's "*.githubusercontent.com"
*  issuer: C=US; O=DigiCert Inc; CN=DigiCert Global G2 TLS RSA SHA256 2020 CA1
*  SSL certificate verify ok.
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://objects.githubusercontent.com/github-production-release-asset-2e65be/595150466/7ebc619a-5088-42b2-b90a-475348892192?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240407%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240407T052031Z&X-Amz-Expires=300&X-Amz-Signature=4e3f6efff3f127efede8c27ef88fb193a51f757ffd3e72bcabc6d925e0247f7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=595150466&response-content-disposition=attachment%3B%20filename%3DDroplr-1.7.0-arm64.dmg&response-content-type=application%2Foctet-stream
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: objects.githubusercontent.com]
* [HTTP/2] [1] [:path: /github-production-release-asset-2e65be/595150466/7ebc619a-5088-42b2-b90a-475348892192?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240407%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240407T052031Z&X-Amz-Expires=300&X-Amz-Signature=4e3f6efff3f127efede8c27ef88fb193a51f757ffd3e72bcabc6d925e0247f7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=595150466&response-content-disposition=attachment%3B%20filename%3DDroplr-1.7.0-arm64.dmg&response-content-type=application%2Foctet-stream]
* [HTTP/2] [1] [user-agent: curl/8.4.0]
* [HTTP/2] [1] [accept: */*]
> GET /github-production-release-asset-2e65be/595150466/7ebc619a-5088-42b2-b90a-475348892192?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240407%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240407T052031Z&X-Amz-Expires=300&X-Amz-Signature=4e3f6efff3f127efede8c27ef88fb193a51f757ffd3e72bcabc6d925e0247f7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=595150466&response-content-disposition=attachment%3B%20filename%3DDroplr-1.7.0-arm64.dmg&response-content-type=application%2Foctet-stream HTTP/2
> Host: objects.githubusercontent.com
> User-Agent: curl/8.4.0
> Accept: */*
> 
< HTTP/2 200 
< content-type: application/octet-stream
< content-md5: U85/YZ6rFH/xD5uvulvXSw==
< last-modified: Mon, 26 Feb 2024 12:37:54 GMT
< etag: "0x8DC36C7C10ECFC8"
< server: Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0
< x-ms-request-id: 87563660-501e-006a-67dd-87397d000000
< x-ms-version: 2020-10-02
< x-ms-creation-time: Mon, 26 Feb 2024 12:37:54 GMT
< x-ms-lease-status: unlocked
< x-ms-lease-state: available
< x-ms-blob-type: BlockBlob
< content-disposition: attachment; filename=Droplr-1.7.0-arm64.dmg
< x-ms-server-encrypted: true
< via: 1.1 varnish, 1.1 varnish
< accept-ranges: bytes
< age: 762
< date: Sun, 07 Apr 2024 05:20:32 GMT
< x-served-by: cache-iad-kcgs7200151-IAD, cache-pao-kpao1770039-PAO
< x-cache: HIT, HIT
< x-cache-hits: 33, 0
< x-timer: S1712467232.032254,VS0,VE311
< content-length: 171537486
< 
{ [32750 bytes data]
* Connection #1 to host objects.githubusercontent.com left intact

2024-04-06 22:20:56 : REQ   : droplr : no more blocking processes, continue with update
2024-04-06 22:20:56 : REQ   : droplr : Installing Droplr
2024-04-06 22:20:56 : INFO  : droplr : Mounting /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.DuHitxN11u/Droplr.dmg
2024-04-06 22:20:59 : DEBUG : droplr : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified   CRC32 $5F8BF416
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified   CRC32 $20C65628
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified   CRC32 $0C4B584E
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified   CRC32 $00000000
Checksumming disk image (Apple_APFS : 4)…
disk image (Apple_APFS : 4): verified   CRC32 $F8BECBA9
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified   CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified   CRC32 $0C4B584E
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified   CRC32 $9BDE6F9B
verified   CRC32 $B87E05A6
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_APFS
/dev/disk5          	EF57347C-0000-11AA-AA11-0030654
/dev/disk5s1        	41504653-0000-11AA-AA11-0030654	/Volumes/Droplr 1.7.0-arm64

2024-04-06 22:20:59 : INFO  : droplr : Mounted: /Volumes/Droplr 1.7.0-arm64
2024-04-06 22:20:59 : INFO  : droplr : Verifying: /Volumes/Droplr 1.7.0-arm64/Droplr.app
2024-04-06 22:20:59 : DEBUG : droplr : App size: 420M	/Volumes/Droplr 1.7.0-arm64/Droplr.app
2024-04-06 22:21:01 : DEBUG : droplr : Debugging enabled, App Verification output was:
/Volumes/Droplr 1.7.0-arm64/Droplr.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: Droplr (MZ25PHMY7Y)

2024-04-06 22:21:01 : INFO  : droplr : Team ID matching: MZ25PHMY7Y (expected: MZ25PHMY7Y )
2024-04-06 22:21:01 : INFO  : droplr : Installing Droplr version 1.7.0 on versionKey CFBundleShortVersionString.
2024-04-06 22:21:02 : INFO  : droplr : App has LSMinimumSystemVersion: 10.13
2024-04-06 22:21:02 : INFO  : droplr : Copy /Volumes/Droplr 1.7.0-arm64/Droplr.app to /Applications
2024-04-06 22:21:03 : DEBUG : droplr : Debugging enabled, App copy output was:
Copying /Volumes/Droplr 1.7.0-arm64/Droplr.app

2024-04-06 22:21:03 : WARN  : droplr : Changing owner to lucas
2024-04-06 22:21:03 : INFO  : droplr : Finishing...
2024-04-06 22:21:06 : INFO  : droplr : App(s) found: /Applications/Droplr.app
2024-04-06 22:21:06 : INFO  : droplr : found app at /Applications/Droplr.app, version 1.7.0, on versionKey CFBundleShortVersionString
2024-04-06 22:21:06 : REQ   : droplr : Installed Droplr, version 1.7.0
2024-04-06 22:21:06 : INFO  : droplr : notifying
displaynotification:7: no such file or directory: /usr/local/bin/dialog
2024-04-06 22:21:06 : DEBUG : droplr : Unmounting /Volumes/Droplr 1.7.0-arm64
2024-04-06 22:21:07 : DEBUG : droplr : Debugging enabled, Unmounting output was:
"disk4" ejected.
2024-04-06 22:21:07 : DEBUG : droplr : Deleting /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.DuHitxN11u
2024-04-06 22:21:07 : DEBUG : droplr : Debugging enabled, Deleting tmpDir output was:
/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.DuHitxN11u/Droplr.dmg
2024-04-06 22:21:07 : DEBUG : droplr : /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.DuHitxN11u
2024-04-06 22:21:07 : INFO  : droplr : Installomator did not close any apps, so no need to reopen any apps.
2024-04-06 22:21:07 : REQ   : droplr : All done!
2024-04-06 22:21:07 : REQ   : droplr : ################## End Installomator, exit code 0
```

BigMacAdmin commented:
Thanks for the contribution.

@lucascantor , is there a good reason that you use `appVersionFromGit` but not `appDownloadURLFromGit` ? Are you certain the download from the developer site and the version on github are never out of sync?

When using a separate source for the `appNewVersion` variable, it has a higher chance of issues.

lucascantor commented:
@BigMacAdmin good question! This is my first contribution, and I didn't know `downloadURLFromGit` was available too. That definitely makes this much simpler.

I've updated my PR with a new commit to use `downloadURLFromGit` too now, alongside `appVersionFromGit` for consistency.

I've also updated my `assemble.sh` output to confirm this works as expected.

BigMacAdmin commented:
Thank you!
